cmd/loadtest/main.go

```diff
@@ -71,12 +86,15 @@ func Run() int {
 		return 1
 	}
 
-	client := horizon.Client{
-		URL:  *horizonDomainFlag,
-		HTTP: &http.Client{Timeout: ClientTimeout},
-	}
+	var clients []horizon.Client
+	for i := 0; i < len(horizonDomainFlags); i++ {
+		client := horizon.Client{
+			URL:  horizonDomainFlags[0],
```

Comment:
guess you wanted to use `[i]` instead of `[0]`?

Reply:
Yes.